lerna.json

```diff
@@ -1,5 +1,4 @@
 {
   "$schema": "node_modules/lerna/schemas/lerna-schema.json",
-  "useWorkspaces": true,
   "version": "7.0.0-alpha.20"
```

Comment:
```suggestion
  "version": "0.0.0-development"
```

Reply:
Both @sequelize/core & @sequelize/validator.js packages have the version set as 7.0.0-alpha.20, is this correct?

Reply:
I think we'll need to update those as well. Our latest alpha is not 20 so it's better to have something marked as development then an old version imo

Reply:
Ok, I've reverted the package version to 0.0.0-development as well as lerna